checkout.sh

```diff
@@ -4,6 +4,20 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-./build-disconnect.py
+if ! hash python; then
+    echo "python is not installed"
+    exit 1
+fi
+
+ver=$(python -V 2>&1 | sed 's/.* \([0-9]\).\([0-9]\).*/\1\2/')
+echo $ver
```

Comment:
I'm not sure it's necessary to print the version to the console, at least not by itself. It could perhaps be more helpful to write the version number, followed by which script it is going to run (based on the version number). Something like:
```Python 27 detected. Runing build-disconnect2.py```